src/wincng.h

```diff
@@ -299,21 +302,20 @@ struct _libssh2_wincng_cipher_ctx {
 struct _libssh2_wincng_cipher_type {
     BCRYPT_ALG_HANDLE *phAlg;
     unsigned long dwKeyLength;
-    unsigned long dwUseIV;
+    int useIV;
+    int ctrMode;
```

Comment:
Bool would be fine by me, I wasn't sure if it was compatible with the C specs libssh2 adheres to, I didn't see boolean being used anywhere.